spec/features/bulk_update_metadata_spec.rb

```diff
@@ -36,8 +38,8 @@
     tds = row.all('td')
     tds[9].find('form > button').click
     # Confirm delete in the popup
-    click_link 'Delete'
-    expect(page).to have_content "Bulk job for APO (#{Settings.default_apo}) deleted."
+    # click_link 'Delete'
+    # expect(page).to have_content "Bulk job for APO (#{Settings.default_apo}) deleted."
```

Comment:
Cruft or useful to keep here?

Reply:
Oops, this wasn't supposed to be in this PR.

Reply:
Removed this stuff